### ISSUE TITLE
Consider creating a Debian package

### ISSUE THREAD

**Issue by olebole** [heliopython/heliopy#415]
Hi,

I am a member of the [Debian Astronomy](https://blends.debian.org/astro) Team. Our aim is to package astronomical software for Debian, from where it will also migrate to derivative distributions like Ubuntu or Mint. Would you consider packaging heliopy for Debian? This would ease the installation of the software and help people keeping it updated. So, it can broaden your user base.

If you would package it, we would be glad to help you when you run into problems. Once the package is ready, we could upload it to the Debian archive. We have a [mailing list](https://lists.debian.org/debian-astro), which I would recommend to subscribe, since this is our main information exchange.

Please feel free to contact me on any questions or comments.

Best regards

Ole

**Comment by dstansby** [comment]
Hello, I definitely would be interested in packaging it, is there a guide somewhere that I can follow?

**Comment by dstansby** [comment]
Closing in favour of #684 , thanks for opening the original request @olebole
